### ISSUE TITLE
Code for De-Normalization

### ISSUE THREAD

**Issue by Emuly** [jaungiers/LSTM-Neural-Network-for-Time-Series-Prediction#60]
Can someone write code for De-Normalization so I can plot predictions on real data values

**Comment by marmartintsang** [comment]
Same request here.

**Comment by RickPang** [comment]
Same request here.

**Comment by ZionYuan** [comment]
Same request here.

**Comment by guillaume-chevalier** [comment]
@alexbrillant We here have the demonstration of a good use-case for a NormalizationWrapper MetaStep object. 

We should code a NormalizationWrapper (MetaStep) in [Neuraxle](https://www.neuraxle.neuraxio.com/stable/examples/boston_housing_regression_with_model_stacking.html) that normalizes the data before sending it to a wrapped pipeline step, and that then denormalizes it before returning the results. 

Inside that, the same normalization could be applied to `y` from the normalization values learned from `x` such that the wrapped neural network just sees normalized data and also predicts normalized data.

The same concept could go with a `LogWrapper` that would take the LogPlusOne of the values and also undo the transformation after. 

Even further into this thinking, we could have a step that instead has a principal wrapped step, and also a list (or pipeline) of other steps to "apply" as preprocessing and then "unapply" as a reverse transformation after the processing is done to properly wrap the neural network. Let's call this a `ReversiblePreprocessingWrapper`.

Actually, I opened an issue here to do that, as we'll need it anyways: https://github.com/Neuraxio/Neuraxle/issues/59

**Comment by kitt-th** [comment]
> Can someone write code for De-Normalization so I can plot predictions on real data values

the denormalization formula is given onthe altumingelligence article.... i could post the code here to do it in place where the normalization is done if you like, but it it very straightforward

**Comment by hao940953596** [comment]
@kitt-th Can you write the code here? I really need it, thank you.

**Comment by kitt-th** [comment]
sure
to do a reverse check - ie denormalisation, inside the normalisation function, do this

denormalized_col = [ (  float(window[0, col_i])  * (float(n)  + 1 ) ) for n in normalised_col ]

(denormalisation func - Pi = Po*(Ni + 1) (from the article text)

**Comment by jimfoo88** [comment]
I don't think it is possible to denormalise the predicted data with that formula. It will results in some values very close to 0.
I suggest to change the normalisation to use MinMaxScaler from sklearn instead. You can denormalise using the invert function.

**Comment by zhangzhang-vvv** [comment]
> I don't think it is possible to denormalise the predicted data with that formula. It will results in some values very close to 0.
> I suggest to change the normalisation to use MinMaxScaler from sklearn instead. You can denormalise using the invert function.

could you write this MinMaxScaler code here？thank you！！
